concordium-consensus/tests/globalstate/Basic/AccountTable.hs

```diff
@@ -5,7 +5,7 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE ViewPatterns #-}
 
-module Concordium.GlobalState.Basic.BlockState.AccountTable where
+module Basic.AccountTable where
```

Comment:
Why are these modules renamed?

Reply:
Since they are only used by some tests now, I moved them out of `src` and into `tests`, later we should remove them entirely, but that will be another PR.